gateway/test/suite_test.go

```diff
@@ -139,6 +139,14 @@ func (s *GatewayOpsTestSuite) missingDocId() string {
 	return s.randomDocId()
 }
 
+func (s *GatewayOpsTestSuite) tooLongDocId() string {
+	id := s.randomDocId()
```

Comment:
Would you be able to make this generate a key that is precisely 1 character longer than is allowed?  This will ensure we capture off-by-one calculation errors.

Reply:
Done